controllers/drplacementcontrol.go

```diff
@@ -307,6 +313,9 @@ func (d *DRPCInstance) RunFailover() (bool, error) {
 		d.setDRPCCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
 			d.getConditionStatusForTypeAvailable(), string(d.instance.Status.Phase), msg)
 
+		rmnutil.ReportIfNotPresent(d.reconciler.eventRecorder, d.instance, corev1.EventTypeWarning,
```

Comment:
We potentially need an event if `isPlacementNeedsFixing` fails, or even if it executed a fixup, so that we know a state was updated/fixed.

Reply:
Done